### ISSUE TITLE
Add healthcheck route

### ISSUE THREAD

**Issue by winston0410** [svenstaro/miniserve#1435]
At the moment, there is no healthcheck route for miniserve server, which is crucial for setting up a readiness probes and an liveness probe in K8S.

Using `/` as healthcheck alternative would not work, as we cannot set auth for healthcheck in K8S.

**Comment by svenstaro** [comment]
I agree this would be nice to have. Would you like to add this? Should be fairly straight-forward.

**Comment by svenstaro** [comment]
I went ahead and implemented this. :)
